torrent.py

```diff
@@ -105,6 +110,26 @@ def ping_tracker(self):
                         self.filewriter,
                     )
                     self.peer_list.append(peer)
+                    
```

Comment:
lines 113 - 135 is essentially allowing us to do many asynchronous tasks concurrently.